components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -130,6 +198,11 @@ service Policies {
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}:members";
     option (chef.automate.api.iam.policy).action = "iam:policyMembers:delete";
     };
+  /*
+  Add policy members
+
+  Add specific members to the member list for a specific policy.
+  */
   rpc AddPolicyMembers (AddPolicyMembersReq) returns (AddPolicyMembersResp) {
```

Comment:
I'm not sure where `AddPolicyMembersResp` is defined but it is not documented in the UI.

Reply:
What specifically are you looking for here? For a lot of these, the request and response woulda just been copy pasta'ed content. Is that what we want?

Reply:
The API docs are a reference so a user will likely seek out a particular endpoint rather than reading the doc top to bottom. If a user seeks out a specific endpoint they will likely not have seen a particular description or explanation if it is elsewhere in the doc.